reflectivity_ui/interfaces/data_manager.py

```diff
@@ -452,6 +452,16 @@ def is_gisans_available(self, active_only=True):
                 return False
         return True
 
+    def reduce_spec(self):
+        """
+        Calculate reflectivity for all runs in the reduction list
+        """
+        for nexus_data in self.reduction_list:
```

Comment:
I guess this is OK since the user doesn't experience any time-lag when using the GUI